app/controllers/my_facilities/ranked_facilities_controller.rb

```diff
@@ -42,13 +42,7 @@ def show
   private
 
   def set_last_updated_at
-    last_updated_at =
-      begin
-        Time.parse(Rails.cache.fetch(Constants::MATVIEW_REFRESH_TIME_KEY))
```

Comment:
We don't need to parse things out of the cache - Rails does a Marshall.dump when setting things in the cache, so when we read it will come out as a `Time` correctly w/ the timezone and everything.